src/App.tsx

```diff
@@ -15,71 +19,99 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = { selectedGood: string };
+
+export class App extends Component<{}, State> {
+  state = { selectedGood: 'Jam' };
+
+  isSelected = (item: string) => (
```

Comment:
from its name it should be clear that it is a function
```suggestion
  checkIsSelected = (item: string) => (
```